ubyssey/templates/components/event-detail.html

```diff
@@ -0,0 +1,36 @@
+<div class="c-event c-event--detail">
+  {% if event.image %}
+    <div class="c-event__image" style="background-image: url('{{ event.image.url }}')"></div>
+  {% endif %}
+
+  <div class="c-event__meta">
+    <div class="c-event__date">
+    {% include 'components/event-date.html' with event=event size="large" %}
+    </div>
+    <div class="c-event__info">
+      <h1 class="c-event__title">{{ event.title|wordwrap:30|linebreaksbr }}</h3>
+    {% if event.start_time|date:"Ymd" == event.end_time|date:"Ymd" %}
+      <p class="c-event__time">{{ event.start_time|date:"l, F jS \a\t g A" }} - {{ event.end_time|date:"g A"}}</p>
+    {% else %}
+      <p class="c-event__time">{{ event.start_time|date:"l, F jS \a\t g A" }} - {{ event.end_time|date:"l, F jS \a\t g A" }}</p>
+    {% endif %}
+      <p class="c-event__location">{{ event.location|wordwrap:40|linebreaksbr }}</p>
+      <p class="c-event__location">Hosted by {{ event.host }}</p>
+    </div>
+  </div>
+
+  <div class="c-event__buttons">
+    {% if event.event_url %}
+      <a class="c-button" href="{{ event.event_url }}">
+        <i class="fa fa-facebook-square"></i>
+        <span>View on Facebook</span>
+      </a>
+    {% endif %}
+    {% if event.ticket_url %}
+      <a class="c-button " href="{{ event.ticket_url }}">
+        <i class="fa fa-ticket" style="vertical-align: middle;"></i>
```

Comment:
I think this style is redundant with with `ubyssey/static/src/styles/components/_button.scss`